ydb/core/cms/node_checkers.h

```diff
@@ -26,6 +27,39 @@ class INodesChecker {
         NODE_STATE_DOWN /* "Down" */
     };
 
+    class TTryToLockResult {
+    public:
+        static TTryToLockResult Success() {
+            return TTryToLockResult(true);
+        }
+
+        static TTryToLockResult Fail(const TReason &reason) {
+            return TTryToLockResult(false, reason);
+        };
+
+        bool IsSuccess() const {
+            return IsSuccess_;
+        }
+
+        TReason GetReason() const {
+            return Reason;
+        }
+
+    private:
+        TTryToLockResult(bool isSuccess, const TReason &reason)
+        : IsSuccess_(isSuccess)
+        , Reason(reason)
+        {}
```

Comment:
Здесь и в других местах поехала верстка:
```suggestion
        TTryToLockResult(bool isSuccess, const TReason &reason)
            : IsSuccess_(isSuccess)
            , Reason(reason)
        {}
```